orbit/pkg/update/runner.go

```diff
@@ -156,6 +160,27 @@ func (r *Runner) UpdateAction() (bool, error) {
 	return didUpdate, nil
 }
 
+func (r *Runner) needsSymlinkUpdate() (bool, error) {
```

Comment:
Nit: This handles the `orbit` target only, so I'd rename this `needsOrbitSymlinkUpdate`.

Reply:
(Or pass the `target string` as argument.)

Reply:
Orbit is the only thing that gets symlinked, AIUI, so I'll just rename